Index: src/pages/EditPlant.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, {useEffect, useState} from 'react';\r\nimport {useHistory} from 'react-router-dom';\r\n\r\n\r\nconst EditPlant = () => {\r\n\r\n    const [ plantInfo, setPlantInfo ] = useState({title:'',image:'',schedule:'',species:'',id:''});\r\n    const history = useHistory();\r\n\r\n    useEffect(() => {\r\n            setPlantInfo(history.location.state.plant)\r\n    }, []);\r\n\r\n\r\n    return (\r\n        <>\r\n            <form>\r\n                <h2>Edit {plantInfo.title}</h2>\r\n                <label htmlFor=\"title\">Name:\r\n                    <input\r\n                        type=\"text\"\r\n                        name=\"title\"\r\n                        value={plantInfo.title}\r\n\r\n                    />\r\n                </label>\r\n            </form>\r\n\r\n        </>\r\n    )\r\n}\r\n\r\nexport default EditPlant;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/EditPlant.js b/src/pages/EditPlant.js
--- a/src/pages/EditPlant.js	(revision 61764c543eccd81f45f6de164394723b978b7bc0)
+++ b/src/pages/EditPlant.js	(date 1624348860371)
@@ -1,29 +1,64 @@
 import React, {useEffect, useState} from 'react';
 import {useHistory} from 'react-router-dom';
+import { useForm} from "../hooks/useForm";
 
 
 const EditPlant = () => {
-
     const [ plantInfo, setPlantInfo ] = useState({title:'',image:'',schedule:'',species:'',id:''});
     const history = useHistory();
+    const {formData, handleInputChange, handleSubmit } = useForm({
+        title: plantInfo.title,
+        image: plantInfo.image,
+        schedule: plantInfo.schedule
+    })
 
     useEffect(() => {
             setPlantInfo(history.location.state.plant)
     }, []);
 
+    console.log("formData:", formData)
 
     return (
         <>
-            <form>
+            <form onSubmit={handleSubmit}>
                 <h2>Edit {plantInfo.title}</h2>
                 <label htmlFor="title">Name:
                     <input
+                        onChange={handleInputChange}
                         type="text"
                         name="title"
-                        value={plantInfo.title}
-
+                        placeholder={plantInfo.title}
+                        value={formData.title}
                     />
                 </label>
+                <label htmlFor="schedule">Schedule:
+                    <input
+                        onChange={handleInputChange}
+                        type="text"
+                        name="schedule"
+                        placeholder={plantInfo.schedule}
+                        value={formData.schedule}
+                    />
+                </label>
+                <label htmlFor="image">image:
+                    <input
+                        onChange={handleInputChange}
+                        type="text"
+                        name="image"
+                        placeholder={plantInfo.image}
+                        value={formData.image}
+                    />
+                </label>
+                <label htmlFor="species">Species:
+                    <input
+                        type="text"
+                        name="species"
+                        placeholder={plantInfo.species}
+                        value={formData.species}
+                        onChange={handleInputChange}
+                    />
+                </label>
+                <button>Submit</button>
             </form>
 
         </>
Index: src/hooks/useForm.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/hooks/useForm.js b/src/hooks/useForm.js
new file mode 100644
--- /dev/null	(date 1624348890776)
+++ b/src/hooks/useForm.js	(date 1624348890776)
@@ -0,0 +1,19 @@
+import {useState,  } from "react";
+
+export const useForm = (initialState = {}) => {
+    const [formData, setFormData] = useState(initialState);
+
+
+    const handleInputChange = (event) => {
+        event.stopPropagation();
+        setFormData({...formData, [event.target.name]: event.target.value});
+    }
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        onsubmit?.(formData)
+    };
+
+
+    return {formData, handleInputChange, handleSubmit, setFormData};
+}
\ No newline at end of file
