Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import axios from 'axios';\r\n\r\nfunction App() {\r\n    axios.get('https://www.dnd5eapi.co/api/monsters')\r\n        .then(response => console.log(response))\r\n        .catch(error => console.log(error))\r\n\r\n    return (\r\n        <div className=\"App\">\r\n            <h1>hello there...</h1>\r\n        </div>\r\n    );\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.js b/src/App.js
--- a/src/App.js	(revision 7234c356deae1cc7d95b8365bfedb455f0934595)
+++ b/src/App.js	(date 1623904955765)
@@ -1,4 +1,7 @@
 import axios from 'axios';
+import {BrowserRouter} from 'react-router-dom';
+import Header from "./components/Header";
+import PlantList from "./pages/PlantList";
 
 function App() {
     axios.get('https://www.dnd5eapi.co/api/monsters')
@@ -6,9 +9,18 @@
         .catch(error => console.log(error))
 
     return (
-        <div className="App">
-            <h1>hello there...</h1>
-        </div>
+        <BrowserRouter>
+            <div className="App">
+                <Header/>
+                <Switch>
+                    <Route {...rest} path={'/PlantList'} render={
+                        ({props})=>{
+                            <PlantList />
+                    } }/>
+                    <Route path={['/','/home']} component={Login}/>
+                </Switch>
+            </div>
+        </BrowserRouter>
     );
 }
 
Index: src/pages/Home.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/Home.js b/src/pages/Home.js
new file mode 100644
--- /dev/null	(date 1623971089171)
+++ b/src/pages/Home.js	(date 1623971089171)
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Home = () => {
+
+}
+
+export default Home;
\ No newline at end of file
