Index: src/pages/EditPlant.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React, {useEffect, useState} from 'react';\r\nimport {useHistory} from 'react-router-dom';\r\n\r\n\r\nconst EditPlant = () => {\r\n\r\n    const [ plantInfo, setPlantInfo ] = useState({title:'',image:'',schedule:'',species:'',id:''});\r\n    const history = useHistory();\r\n\r\n    useEffect(() => {\r\n            setPlantInfo(history.location.state.plant)\r\n    }, []);\r\n\r\n\r\n    return (\r\n        <>\r\n            <form>\r\n                <h2>Edit {plantInfo.title}</h2>\r\n                <label htmlFor=\"title\">Name:\r\n                    <input\r\n                        type=\"text\"\r\n                        name=\"title\"\r\n                        value={plantInfo.title}\r\n\r\n                    />\r\n                </label>\r\n            </form>\r\n\r\n        </>\r\n    )\r\n}\r\n\r\nexport default EditPlant;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/pages/EditPlant.js b/src/pages/EditPlant.js
--- a/src/pages/EditPlant.js	(revision 61764c543eccd81f45f6de164394723b978b7bc0)
+++ b/src/pages/EditPlant.js	(date 1624330086210)
@@ -21,7 +21,6 @@
                         type="text"
                         name="title"
                         value={plantInfo.title}
-
                     />
                 </label>
             </form>
